EIPS/eip-7928.md

```diff
@@ -93,9 +93,9 @@ The `code` field in `AccountAccess` is optional and MUST be present only when a
 
 For reads, each `SlotAccess` MUST contain an empty `accesses` list.
 
-The `BalanceDiffs` structure tracks every address with a balance change, including transaction senders, recipients, and the block's coinbase address. 
+The `BalanceDiffs` structure tracks every address with a balance change, including transaction senders, recipients, and the block's coinbase address. Touched accounts without balance changes MUST be omitted.
 Each entry MUST include the transaction index and the signed balance delta per address for each transaction.
-11 bytes are sufficient to represent the total ETH supply.
+12 bytes are sufficient to represent the total ETH supply.
```

Comment:
Maybe for completeness `2^(12*8) / 1e18 / 1e6` (`1e18` = 1 eth) = approx 79228 million ETH (which is much higher than current supply of ~121 million.

Also note: we have to encode the balance diffs as integer. So one of the bits is used as sign in the balance diffs. For 11 bytes we thus have `11*8-1 = 87` bits, which yields `2^87/1e18/1e6` = 154 million. Which is close-ish to the total supply currently. (Of course in practice we will not see a balance diff anywhere of the total supply, but if we are well above it then we can state that balance diffs thus always fit, without exceptions)